README.md

```diff
@@ -57,14 +57,6 @@ A large collection of showcase projects are included in [awesome-docsify](https:
 - [Awesome docsify](https://github.com/docsifyjs/awesome-docsify)
 - [Community chat](https://discord.gg/3NwKFyR)
 
-## Similar Projects
-
-| Project                                          | Description                              |
-| ------------------------------------------------ | ---------------------------------------- |
-| [Docusaurus](https://docusaurus.io) | Docusaurus makes it easy to maintain Open Source documentation websites |
-| [MkDocs](https://www.mkdocs.org) | Project documentation with Markdown |
-| [VitePress](https://vitepress.dev/) | Vite & Vue Powered Static Site Generator |
-
 ## Contributing
```

Comment:
Resolving the conflict and remove this selection directly since we may not need individual ads here :)